core/src/consensus/tendermint/worker.rs

```diff
@@ -1044,7 +1044,13 @@ impl Worker {
             return Seal::None
         }
 
-        assert!(self.is_signer_proposer(&parent_hash));
+        // We don't know at which view the node starts generating a block.
+        // If this node's signer is not proposer at the current view, return none.
+        if self.is_signer_proposer(&parent_hash) {
+            cwarn!(ENGINE, "Proposer is generated after view change");
```

Comment:
```suggestion
            cwarn!(ENGINE, "View is changed after requesting the seal");
```

Reply:
View is not changed after the seal request. View is changed after the block generation request.
What do you think about "Seal request for an old view"?

Reply:
That's good.